code/modules/vehicles/wheelchair.dm

```diff
@@ -125,3 +120,43 @@
 	max_integrity = 200
 	armor = list(MELEE = 20, BULLET = 20, LASER = 20, ENERGY = 0, BOMB = 20, BIO = 0, RAD = 0, FIRE = 30, ACID = 40)
 	custom_materials = list(/datum/material/gold = 10000)
+	foldabletype = /obj/item/wheelchair/gold
+
+/obj/item/wheelchair
+	name = "wheelchair"
+	desc = "A collapsed wheelchair that can be carried around."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "wheelchair_folded"
+	inhand_icon_state = "wheelchair_folded"
+	lefthand_file = 'icons/mob/inhands/items_lefthand.dmi'
+	w_class = WEIGHT_CLASS_NORMAL
+	force = 8 //Force is same as a chair
+	var/unfolded_type = /obj/vehicle/ridden/wheelchair/
+
+/obj/item/wheelchair/gold
+	name = "gold wheelchair"
+	desc = "A collapsed, shiny wheelchair that can be carried around."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "wheelchair_folded_gold"
+	force = 10
+	unfolded_type = /obj/vehicle/ridden/wheelchair/gold
+
+/obj/vehicle/ridden/wheelchair/MouseDrop(over_object, src_location, over_location)  //Lets you collapse wheelchair
+	. = ..()
+	if(over_object == usr && Adjacent(usr) && can_fold)
+		if(!ishuman(usr) || !usr.canUseTopic(src, BE_CLOSE))
+			return FALSE
+		if(has_buckled_mobs())
+			return FALSE
+		usr.visible_message("<span class='notice'>[usr] collapses \the [src.name].</span>", "<span class='notice'>You collapse \the [src.name].</span>")
+		var/obj/vehicle/ridden/wheelchair/B = new foldabletype(get_turf(src))
+		usr.put_in_hands(B)
+		qdel(src)
+
+/obj/item/wheelchair/attack_self(mob/user)  //Deploys wheelchaair on in-hand use
+	deploy_wheelchair(user, user.loc)
+
+/obj/item/wheelchair/proc/deploy_wheelchair(mob/user, atom/location)
+	var/obj/vehicle/ridden/wheelchair/R = new unfolded_type(location)
+	R.add_fingerprint(user)
+	qdel(src)
```

Comment:
This will clean the object, remove radiation, acid, fingerprints, you name it.
Any chance that could be addressed?